rpc/frmops.c

```diff
@@ -496,6 +496,7 @@ static void item_sent(struct m0_rpc_item *item)
 		 */
 		M0_ASSERT(m0_rpc_item_is_request(item));
 		M0_ASSERT(item->ri_error == -ECANCELED);
+		m0_rpc_item_sent_invoke(item);
```

Comment:
Do we need to update `rio_sent()` documentation about this?

Reply:
> Do we need to update `m0_rpc_item_ops::rio_sent()` documentation about this?

The documentation already covers this: "or item sending failed in which case item->ri_error != 0". It was rpc implementation that was inconsistent with the documentation.

Reply:
Ok.